aptos-move/framework/aptos-framework/sources/event.move

```diff
@@ -54,4 +54,23 @@ module aptos_framework::event {
     public fun destroy_handle<T: drop + store>(handle: EventHandle<T>) {
         EventHandle<T> { counter: _, guid: _ } = handle;
     }
+
+    #[test_only]
+    use std::option::{Self, Option};
+
+    #[test_only]
+    public fun emitted_event<T: drop + store>(handle: &EventHandle<T>, sequence_number: u64): Option<T> {
```

Comment:
Cant this be named something more descriptive like `get_emitted_event()` ?

Reply:
I think the desire for "get_" prefix is a personal choice.

Reply:
This doesn't fit in with the model we landed on.